crates/matrix-sdk/src/event_cache/deduplicator.rs

```diff
@@ -267,4 +267,33 @@ mod tests {
             assert!(events.next().is_none());
         }
     }
+
+    #[test]
+    fn test_bloom_filter_growth() {
+        // This test was used as a testbed to observe, using `valgrind --tool=massive`,
+        // the total memory allocated by the deduplicator. We keep it checked in
+        // to revive this experiment in the future, if needs be.
+
+        let num_rooms = 10;
+        let num_events = 100;
+        //let num_rooms = std::env::var("ROOMS").unwrap().parse().unwrap();
```

Comment:
Can't we use `unwrap_or(100)` here so we don't need to modify the test to rerun the valgrind test?

Reply:
Good idea, done something equivalent, thanks.